pulsar-metadata/pom.xml

```diff
@@ -94,6 +94,23 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.gaul</groupId>
+        <artifactId>modernizer-maven-plugin</artifactId>
+        <version>${maven-modernizer-plugin.version}</version>
+        <configuration>
```

Comment:
is it better to set the configuration in the root POM and share it with all submodules

cc @MarvinCai

Reply:
yeah, we do that in mainline branch, this one might need a rebase.

Reply:
Fixed review comments. Could you please review.